xml/System.Diagnostics/ActivityCreationOptions`1.xml

```diff
@@ -101,8 +103,8 @@
         <ReturnType>T</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the parent associated with the activity.</summary>
```

Comment:
>Gets the parent associated with the activity. [](start = 17, length = 45)

The parent context or Parent Id that the Activity will get create with.

Reply:
```suggestion
        <summary>Gets the parent context or parent Id that the activity will get created with.</summary>
```